docs/quick-start.md

```diff
@@ -60,6 +60,8 @@ For more in depth Docker instructions, follow the [Docker environment for Jetpac
 
 In order to test features that require a WordPress.com connection and other network related Jetpack features, you'll need a test site that can create local HTTP tunnels. If you're an Automattician, we recommend using Jurassic Tube:
 
+**Warning: This creates a tunnel to your local machine. Only `jetpack docker jt-up` when needed for testing and `jetpack docker jt-down` when completed.**
```

Comment:
Let's be a bit more clear for those who may not know the implication of what that means. 
```suggestion
**Warning: This creates a tunnel to your local machine which means that if it is compromised, so is everything on your computer. Only `jetpack docker jt-up` when needed for testing things that require the site to be publicly accessible, and `jetpack docker jt-down` when completed.**
```